README.md

```diff
@@ -39,7 +39,13 @@ In production, printed JSON will look like this (except that it's not pretty pri
 
 ### Express integration
 
-If you're running an express server, you can register the logger middleware to log HTTP requests.
+If you're running an express server, you can register the logger middleware to
+log HTTP requests. GraphQL requests get automatically detected and attached as
+`meta.graphql`, with properties `operationName` and the respective `variables`.
+To limit the size of the pyload for the logged variables, you can pass a
```

Comment:
A few things to clean up here:
- typo: "pyload"
- `createLoggerMiddleware({logger, maxGraphQLVariablesLength=128})` isn't valid JS syntax.
- The `-1` case is not documented

What about including a paragraph about `maxGraphQLVariablesLength` after the sample code?